packages/dataviews/src/view-grid.js

```diff
@@ -43,14 +43,14 @@ export default function ViewGrid( { data, fields, view, actions, getItemId } ) {
 					className="dataviews-view-grid__card"
 				>
 					<div className="dataviews-view-grid__media">
-						{ mediaField?.render( { item, view } ) }
```

Comment:
We can access the `view` in the fields API directly, no need to pass it to the field through `render`. Same for `view-table.js`. This change is not necessary for this PR to work.